src/golang/cmd/server/queries/postgres.go

```diff
@@ -1,9 +1,69 @@
 package queries
 
+import (
+	"context"
+	"fmt"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/operator"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/database/stmt_preparers"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/google/uuid"
+)
+
 type postgresReaderImpl struct {
 	standardReaderImpl
 }
 
 func newPostgresReader() Reader {
 	return &postgresReaderImpl{standardReaderImpl{}}
 }
+
+func (r *postgresReaderImpl) GetLoadOperatorSpecByOrganization(
```

Comment:
changes `->>` operator here to use `json_extract_path_text`: https://github.com/aqueducthq/aqueduct/blob/main/src/golang/cmd/server/queries/standard.go#L16